src/wasm/kotlin/src/arcs/Particle.kt

```diff
@@ -24,13 +24,13 @@ abstract class Particle : WasmObject() {
     fun connectHandle(handleName: String, willSync: Boolean): Handle? {
         log("Connect called internal '$handleName'")
 
-        if (handles.containsKey(handleName)) {
-            val handle = handles[handleName]!!
+        handles[handleName]?.let {
             if (willSync) {
```

Comment:
The braces could be removed from this:

if (willSync) toSync.add(it)

Reply:
Is there an agreed upon Google code-style for Kotlin? Google code style for Java typically requires braces, in part because of heartbleed type bugs .

Reply:
Yeah, it's at [go/kotlin-style](http://go/kotlin-style)